src/logged-evaluation.nix

```diff
@@ -16,9 +16,10 @@ let
   imported =
```

Comment:
Let's get nixpkgs-fmt to be part of CI, and fail the build if any `*.nix` file isn't formatted properly. (`find . -name '*.nix' | xargs nixpkgs-fmt` I can help more with this)

Reply:
I've added `nix/fmt.sh` which does essentially this, but excludes generated nix files.